Implemented sarsa

Shreya301 opened the pull request.

Shreya301 committed:
Implemented sarsa

vrn25 commented:
Well done! The results look good, except the number of episodes it takes for converging is about 100 times more than given in Sutton and Barto. Can you try making it closer?

Shreya301 committed:
Changes made to windy gridworld and cliffwalking implemented

Shreya301 commented:
@vrn25  Hey! In the windy grid-world I changed the alpha value to 0.3 and it seems to be converging in the same number of episodes as in the book.
I also added my implementation of the cliff-walking environment using sarsa and Q-learning. Let me know if those are fine

vrn25 commented:
Nice work!
